wordle-web/pages/game.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <v-card class="mx-auto my-8" elevation="16" max-width="344">
+    <v-card-item>
+      <v-card-title> Hurdle Over This Wordle!!! </v-card-title>
+      <v-card-subtitle> <strong class="text-red-accent-4">hurdle it. I dare you. </strong> </v-card-subtitle>
+    </v-card-item>
+
+    <v-text-field
+      v-model="input"
+      label="Will you hurdle? (y/n):"
+      outlined
+      clearable />
+    <v-card-actions>
+      <v-spacer />
+      <v-btn
+        color="blue"

```

Comment:
maybe change the button to red or that amber color to match or vice versa (change text to match blue button)

Reply:
I changes it to red